test/itkMockMontageTestOMC.cxx

```diff
@@ -0,0 +1,80 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkMockMontageHelper.hxx"
+
+constexpr unsigned xMontageSize = 3;
+constexpr unsigned yMontageSize = 3;
+using PositionTableType = std::array<std::array<PointType, xMontageSize>, yMontageSize>;
+using FilenameTableType = std::array<std::array<std::string, xMontageSize>, yMontageSize>;
```

Comment:
We want these to be declared inside the main testing method because the test driver will build everything together, and there may be conflicts.